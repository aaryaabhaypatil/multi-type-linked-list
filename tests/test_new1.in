NEW a
NEW too many
NEW 3
asd
asd
asd
NEW 0
NEW -1