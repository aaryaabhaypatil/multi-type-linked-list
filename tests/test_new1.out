INVALID COMMAND: NEW
INVALID COMMAND: NEW
List 0: asd -> asd -> asd
INVALID COMMAND: NEW
INVALID COMMAND: NEW
